test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperAttributeTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    public class TagHelperAttributeTest
+    {
+        public static TheoryData CopyConstructorData
+        {
+            get
+            {
+                // readOnlyTagHelperAttribute
```

Comment:
Seems kinda hard to maintain if the test methods get refactored. If you really want to keep it, you could throw in a `<see cref="..."` instead. But for a single argument having any comment seems unnecessary.